frontend/src/features/idir/search/table/PermitSearchResultColumnDef.tsx

```diff
@@ -25,21 +29,23 @@ export const PermitSearchResultColumnDef = (
       const permit = props.row.original as PermitListItem;
       const { permitId, permitStatus, expiryDate, companyId } = permit;
 
+      const isPermitVoided = (permitStatus: PermitStatus) =>
+        permitStatus === PERMIT_STATUSES.VOIDED;
+
+      console.log(isPermitVoided(permitStatus));
+
       return (
         <>
           <CustomActionLink
-            onClick={() =>{
-              viewPermitPdf(
-                companyId,
-                permitId,
-                () => onDocumentUnavailable(),
-              );
-            }  
-          }
+            onClick={() => {
+              viewPermitPdf(companyId, permitId, () => onDocumentUnavailable());
+            }}
           >
             {props.cell.getValue()}
           </CustomActionLink>
-          {hasPermitExpired(expiryDate) ? (
+          {isPermitVoided(permitStatus) ? (
```

Comment:
And instead of creating this nested ternary statement, simply use the "getDisplayedPermitStatus" as:
```<PermitChip permitStatus={getDisplayedPermitStatus(permitStatus, expiryDate)} />```

Modify the type of PermitChip's permitStatus prop if needed